plenum/server/validator_info_tool.py

```diff
@@ -585,7 +585,9 @@ def __node_info(self):
                     "Last_complete_view_no": self._prepare_for_json(
                         self._node.view_changer.last_completed_view_no),
                     "IC_queue": self._prepare_for_json(
-                        self._get_ic_queue())
+                        self._get_ic_queue()),
+                    "VCDone_queue": self._prepare_for_json(
```

Comment:
I believe we need to revert back the tests for Validator Info checking for this field.